working-groups.en.md

```diff
@@ -76,7 +76,7 @@ Belarussian Academy of Music\
 Internationulnaja, 30\
 BL-220030 Minsk\
 Email: [shapiro@open.by](mailto:shapiro@open.by)\
-Website: [http://www.bgam.edu.by/eng/](http://www.bgam.edu.by/eng/){:target="_blank"}
+Website: [https://www..by/eng/](https://www.bgam.by/eng/){:target="_blank"}
```

Comment:
This is probably not correct...

Reply:
Ok. What should I do? I have other things on that page that I need to fix as well.

Reply:
You can just fix it... by 'not correct' I mean that you're just missing 'bgam' in the URL... there are two dots.

Reply:
Ok, I am about to commit changes for this (thanks for catching) plus some other things on the page.

Reply:
👍